lib/generators/release.rb

```diff
@@ -27,6 +27,10 @@ def before_generation
         patch 'Gemfile' do |contents|
           contents.sub(/gem "sdoc".*/, "gem \"sdoc\", \"~> 2.6.4\"")
         end
+
+        patch 'guides/assets/stylesrc/components/_code-container.scss' do |contents|
+          contents.sub(/^\s+overflow: scroll;/, "")
```

Comment:
The culprit overflow is the only one in the file https://github.com/rails/rails/blob/529f933fc8b13114d308dd0752f76a9e293c8537/guides/assets/stylesrc/components/_code-container.scss#L52